templates/definition/charger/lg-therma.yaml

```
@@ -0,0 +1,69 @@
+template: lg-therma
+products:
+  - brand: LG
+    description:
+      generic: Therma V R290 Monobloc (SG Ready)
+group: heating
+requirements:
+  description:
+    de: |
+      Modbus Verbindung zur LG Therma V Wärmepumpe.
+      Funktioniert über SG Ready Energiezustände für intelligente Steuerung.
+      Inneneinheit/Fernbedienung in den erweiterten Einstellungen > Konnektivität > Energiezustand > Signaltyp > auf "Modbus" einstellen.
+      Da die interne Energiemessung zu hohe Werte anzeigt ist eine externe Energiemessung empfohlen.
+      Werkseinstellung Modbus Addresse(HEX) = 21 (33)
+    en: |
+      Modbus connection to LG Therma V heat pump.
+      Uses SG Ready energy states for intelligent control.
+      Set the indoor unit in the advanced setting > Connectivty > Energystates > Singal Type > to "Modbus".
+      Since the internal power measurement shows to high values, an external power measurement is suggested.
+      Factory setting Modbus (HEX) = 21 (33)
+params:
+  - name: modbus
+    choice: ["rs485"]
+    id: 33
+  - name: tempsource
+    choice: ["warmwater", "flow_temperature"]
+render: |
+  type: sgready
+  setmode:
+    source: map
+    values:
+      1: 7 # Energiesparen
+      2: 2 # Normalbetrieb
+      3: 6 # Boost
+    set:
+      source: modbus
+      {{- include "modbus" . | indent 4 }}
+      register:
+        address: 9
+        type: writeholdings
+        encoding: uint16
+  getmode:
+    source: map
+    values:
+      7: 1 # Energiesparen
+      2: 2 # Normalbetrieb
+      6: 3 # Boost
+    get:
+      source: modbus
+      {{- include "modbus" . | indent 4 }}
+      register:
+        address: 9
+        type: holding
+        encoding: uint16        
+  temp:
+    source: modbus
+    {{- include "modbus" . | indent 2 }}
+    scale: 0.1
+    register:
+      address:  {{ if eq .tempsource "warmwater" -}} 5 {{ else }} 2 {{- end }} # 5 Trinkwasser, 2 Wassereinlasstemperatur
```

Adapt to suggested manfucaturer description for tempsource
```suggestion
      address:  {{ if eq .tempsource "DHW tank water temperature" -}} 5 {{ else }} 2 {{- end }} # 5 DHW Wasser Temperatur, 2 Wassereinlasstemperatur
```